static/src/javascripts/projects/common/modules/ui/welcomeBanner.js

```diff
@@ -0,0 +1,48 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/utils/load-css-promise'
+], function (
+    fastdom,
+    $,
+    storage,
+    template,
+    loadCssPromise
+) {
+    /**
+     * Rules:
+     * 1st visit replace nav bar with banner
+     */
+    var header = document.getElementById('header'),
+        message1 = '<div class="banner-message"><%=HTML%></div>',
+        DATA = {
```

Comment:
No good reason, I was using `smartAppBanner.js` as an example. I'll change this to lower case.